public/api/swagger.json

```diff
@@ -0,0 +1,838 @@
+                    "type": "string",
+                    "format": "url"
+                },
+                "start_price": {
+                    "type": "integer",
+                    "title": "Start Price",
+                    "description": "The starting price of the auction. This is the exclusive ceiling of all bids. For sealed-bid auctions, the highest allowed bid is the starting price minus $1. For open reverse auctions, bidding starts at this amount, but any new bid must be lower than the previous lowest bid.",
+                    "maximum": 15000,
+                    "minimum": 1
+                },
+                "started_at": {
+                    "title": "Start Time",
+                    "description": "The date-time the auction was or is open for bidding. This may be in the future if the auction has not yet started.",
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "ended_at": {
+                    "title": "End Time",
+                    "description": "The date-time the auction is closed and the winning bid is determined.",
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "created_at": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "updated_at": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "winning_bid": {
+                    "title": "Winning Bid",
+                    "description": "The current winning bid for the auction. For open reverse auctions, these fields will provide information about the current lowest bid during the auction. For sealed-bid auctions, these values will be nil until the auction is over and the final winning bid has been determined. If there is no bid, these fields will be nil.",
+                    "type": "object",
+                    "required": [
+                        "amount",
+                        "bidder_id"
+                    ],
+                    "properties": {
+                        "amount": {
+                            "type": [
+                                "integer",
+                                "null"
+                            ]
+                        },
+                        "bidder_id": {
+                            "type": [
+                                "integer",
+                                "null"
+                            ]
+                        }
+                    }
+                },
+                "bids": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Bid"
+                    }
+                }
+            }
+        },
+        "Bid": {
```

Comment:
we should decide on what type of case we're gonna use in our API - I think we've been using snake case in past, although I know some people say JSON should be camel case.